gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -754,86 +757,91 @@ gst_tensor_filter_configure_tensor (GstTensorFilter * self,
    * Check configuration from caps.
    * If true, fully configured tensor info from caps.
    */
-  if (gst_tensors_config_validate (&in_config)) {
-    /** if set-property called and already has info, verify it! */
-    if (prop->input_meta.num_tensors > 0) {
-      if (!priv->combi.in_combi_defined) {
-        if (!gst_tensors_info_is_equal (&in_config.info, &prop->input_meta)) {
-          GST_ERROR_OBJECT (self, "The input tensor is not compatible.");
-          gst_tensor_filter_compare_tensors (&in_config.info,
-              &prop->input_meta);
-          goto done;
-        }
-      }
-    } else {
-      gst_tensors_info_copy (&prop->input_meta, &in_config.info);
-    }
+  if (!gst_tensors_config_validate (&in_config)) {
+    GST_ERROR_OBJECT (self, "Invalid caps, failed to configure input info.");
```

Comment:
I wonder whether `gst_tensors_config_validate` really configures input info.

Reply:
cannot understand, if dimension and data type is invalid here, gst_tensors_config_validate() will return false.

Reply:
Ah, my bad. I thought that the log message is not directly related to `gst_tensors_config_validate`. But, it seems to include the overall function behavior.